webview-ui/src/components/settings/ApiOptions.tsx

```diff
@@ -1258,8 +1264,43 @@ const ApiOptions = ({
 					</>
 				)}
 
+			{selectedProvider === "anthropic" && selectedModelId === "claude-3-7-sonnet-20250219" && (
```

Comment:
Not proud of this; it would be nice if the `ModelInfo` gave a hint instead of hardcoding the name.